charts/ratify/README.md

```diff
@@ -43,7 +43,8 @@ $ helm upgrade -n gatekeeper-system [RELEASE_NAME] ratify/ratify
 | replicaCount                                       | The number of Ratify replicas in deployment                                                                                                                                                                                         | 1                              |
 | affinity                                           | Pod affinity for the Ratify deployment                                                                                                                                                                                              | `{}`                           |
 | tolerations                                        | Pod tolerations for the Ratify deployment                                                                                                                                                                                           | `[]`                           |
-| notationCert                                       | Public certificate/certificate chain used to create inline certstore used by Notation verifier.                                                                                                                                     | ``                             |
+| notationCert                                       | Public certificate/certificate chain used to create inline certstore used by Notation verifier. This value will be ***deprecated*** in Ratify 1.3 release. Please switch to using notationCerts to specify an array of verification certificates                                                                                                                                  | ``                             |
```

Comment:
I might miss the related discussion, did we make the decision to deprecate it in 1.3 release?

Reply:
This is value is deprecated, and will be removed in future releases for Ratify

Reply:
fixed.